src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -605,6 +605,11 @@ public async Task<IActionResult> OptOutStandardVersion(string referenceNumber, s
         [ModelStatePersist(ModelStatePersist.Store)]
         public async Task<IActionResult> OptOutStandardVersion(OptOutStandardVersionViewModel model)
         {
+            if (!ModelState.IsValid)
```

Comment:
On line 613 the model is checked for null but prior to that the model properties are used - if a validator is being used the model being null should be checked there instead

Reply:
I didn't mean checked as a validation rule I meant check and then throws an exception

Reply:
This code snippet hasn't updated in the Conversations tab of GitHub, but it's been changed. 

During our discussions I believe we agreed to throw in the controller and validate in the validator. This would keep in consistent with the dozens of other endpoints, plus I'm not sure if a validator should be throwing.

Reply:
Agreed